### ISSUE TITLE
Create a test which can be used for different steppers with interpolation

### ISSUE THREAD

**Issue by jonapost** [jonapost/field_propagation#5]
The test should have the following methods: 
- check_step() method which makes a step and checks it,  
- check_interpolation() 
- exact_solution functor which provides the exact solution (or a good enough approximation.)

**Comment by somnergy** [comment]
This can be done with codes of various tests previously implemented. I am thinking of designing a class that can handle this. Do you want to see <em> MagInt_Driver </em> being used in this ? Or else there won't be error control (unless it is implemented explicitly without <em> MagInt_Driver </em> ). Of course, if we want to test several steppers against each other for performance, error control will be necessary.

**Comment by jonapost** [comment]
Yes, please do design a class which implements this test.  Then it can be applied for different steppers very easily.
I agree that using MagInt_Driver makes sense - there is no point in interpolating steps which are rejected.

**Comment by somnergy** [comment]
<a href=https://github.com/jonapost/field_propagation/commit/12b301a0eeb1a67871870743374a2ef5f3046231> RKTest_Beta_2 </a> committed
